app/main/RTMedia.php

```diff
@@ -343,30 +343,30 @@ function custom_style_for_activity_image_size() {
 
 	function custom_style_for_gallery_image_size() {
 		// Get width from rtMedia settings.
-		$width  = ( isset( $this->options['defaultSizes_photo_medium_width'] ) ) ? $this->options['defaultSizes_photo_medium_width'] : '0';
+		$width  = ( isset( $this->options['defaultSizes_photo_thumbnail_width'] ) ) ? $this->options['defaultSizes_photo_thumbnail_width'] : '0';
 		// Get height from rtMedia settings.
-		$height = ( isset( $this->options['defaultSizes_photo_medium_height'] ) ) ? $this->options['defaultSizes_photo_medium_height'] : '0';
+		$height = ( isset( $this->options['defaultSizes_photo_thumbnail_height'] ) ) ? $this->options['defaultSizes_photo_thumbnail_height'] : '0';
 
 		$media_height = $height . 'px';
 		$media_width  = $width . 'px';
 
 		// If height or width given zero, then show it's original height or with.
 		if ( '0' === $height ) {
```

Comment:
this condition never will be true
if there is no `defaultSizes_photo_thumbnail_height` set then it will take `0`
Line [351](https://github.com/rtMediaWP/rtMedia/pull/994/files#diff-f23e65e07ebeccfcd8de51efd0419621R350) makes it `0px`

same goes for width

Reply:
@milindmore22 At line 351, I assigned a value to a different variable `$media_height` which will become `0px`. `$height` will remain `0` only.

Let me know if I misunderstood something.

Reply:
ahh sorry, my bad